analyzer/analyzer.go

```diff
@@ -58,6 +58,7 @@ func New() *analysis.Analyzer {
 	n := newPerfSprint()
 	r := &analysis.Analyzer{
 		Name:     "perfsprint",
+		URL:      "https://github.com/catenacyber/perfsprint",
 		Doc:      "Checks that fmt.Sprintf can be replaced with a faster alternative.",
```

Comment:
BTW @alexandear @mmorel-35 @catenacyber Do you have a better text to suggest for this field ?

Here is mine

```suggestion
		Doc:      "Checks instances of fmt.Sprintf, fmt.Sprint, and fmt.Errorf that could be replaced with more performant alternatives.",
```

Reply:
Faster is moré precise than moré performant which could mean use less memory

Reply:
It was just a suggestion I dropped in the PR.

I wanted a feedback about mentioning fmt.Errorf at least.

But it's not in the PR, let's resolve